packages/storybook/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.52.18
+------------------------------
+*June 22, 2022*
+
+### Changed
+- `vue.config` to include `f-status-banner` in components using `@use` and `@forward`.
+
+
```

Comment:
I think there is a version missing here. it skipped `.17`

Reply:
Yeah you're right - this may have been a mistake during a rebase